rust/xaynet-analytics/src/database/common.rs

```diff
@@ -111,3 +144,36 @@ where
         )
     }
 }
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct RelationalField {
+    pub value: String,
+    pub collection_name: String,
+}
+
+// NOTE: when split_once gets to stable, it would be a much better solution for this
+// https://doc.rust-lang.org/std/string/struct.String.html#method.split_once
+impl From<&str> for RelationalField {
```

Comment:
Since this can fail would be better to implement `TryFrom` instead.

Reply:
makes sense. done